### DISCUSSION TITLE
Where are we in 2023 ?

### DISCUSSION THREAD

**Discussion by masterleo** [phil-barrett/grblHAL-teensy-4.x#80] [General]
Hi just to know, why  has there not been an update to the github pages since 2020 ? Is this still active and usefull ? 
I would like to use grbl hal, is this also good ? https://expatria.myshopify.com/products/flexi-hal ?

**Comment by phil-barrett** [comment]
Actually, there have been several updates here in the last year. Yes, the Teensy based breakout board is still available [tindie](https://www.tindie.com/products/philba/grblhal-bob-unkit-for-teensy-41-t41u5xbb/) and the [main page is here](https://www.grbl.org/grblhal-breakout-board).  grblHAL is very active.  I have a new board based on the Raspberry Pi Pico coming out soon.

**Comment by masterleo** [comment]
Ok I see, i was getting worried because of this : 
<img width="1086" alt="image" src="https://github.com/phil-barrett/grblHAL-teensy-4.x/assets/8157302/66484826-a5c7-4cec-b688-5074fec18030">
It is writen "last change in 2020"  on pins.h and board layout so... A bit old ?

**Comment by masterleo** [comment]
Can I place an order now as you said ? the shop is open I guess ?

**Reply by phil-barrett** [reply]
Yes.  Use the [Tindie store](https://www.tindie.com/products/philba/grblhal-bob-unkit-for-teensy-41-t41u5xbb/). I've sold over 1000 boards so far and it's being used in a lot of different machines.  Also, the [grblHAL web builder](http://svn.io-engineering.com:8080/) is used to configure it, no need to build from source any more.

The design was done in 2020.  I will eventually update the Teensy BOB design though have been more focused on the Pico based design. Am putting it up for sale today.  Here's the [GitHub page for it](https://github.com/phil-barrett/PicoCNC).
